### PR TITLE
Fixes #15802- Enables turning off SSL validation for Repo URLS

### PR DESCRIPTION
**Pull request by parthaa**

### CODE REVIEW

**Comment by ehelms** [comment]
Do we want users turning this off for Red Hat content?

**Comment by parthaa** [comment]
> Do we want users turning this off for Red Hat content?

Ideally not. Its mostly for docker repositories sometimes. Not meant to be used in a production setup.

**Comment by parthaa** [comment]
Added Verify SSL on Sync for new repo page
New Repo

Displays Verify SSL yes/no
Repo Details

Pulp admin should show the "SSL Validation" fields set accordingly
Pulp Admin Yes Repo Details


Changing the Verify SSL to no should get indicated appropriately
Repo Details NO

Pulp admin should show the "SSL Validation" fields set accordingly
Pulp Admin NO Repo Details

hammer repository create --help should show the new "--verify-ssl-on-sync" flag
Hammer

**Comment by parthaa** [comment]
@ehelms @jlsherrill I just need to regenerate cassettes to make the build green. Which I 'll do if you guys are satisfied with the code/

File: db/migrate/20160722193256_add_verify_ssl_on_sync_to_repository.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class AddVerifySslOnSyncToRepository < ActiveRecord::Migration
+  def up
+    add_column :katello_repositories, :verify_ssl_on_sync, :boolean, :default => true
```
Comment (jlsherrill) [inline db/migrate/20160722193256_add_verify_ssl_on_sync_to_repository.rb]: `:null => false` ?

**Comment by jlsherrill** [comment]
One small comment, overall looks good.  Didn't find any issues

**Comment by parthaa** [comment]
[test]

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**